### DISCUSSION TITLE
a1ive's member account on reboot.pro

### DISCUSSION THREAD

**Discussion by Taviruni** [chenall/grub4dos#330] [General]
I saw a post from you in reboot.pro forum commenting you were not able to edit your posts.

I asked **Brito** the forum administrator to fix that on this [post](http://reboot.pro/index.php?showtopic=22539#entry219221).

Now your account was promoted to developer, and you are able to edit your posts and also attach some files on them.

alacran (from reboot.pro)

**Comment by a1ive** [comment]
Thanks. I've already updated g4e download link on my post.
PS. my name is a1ive, not a1ve.

**Reply by Taviruni** [reply]
Sorry for the typo, it is fixed now.
